extra/mariabackup/xtrabackup.cc

```diff
@@ -1194,23 +1194,19 @@ struct my_option xb_client_options[]= {
      GET_STR, REQUIRED_ARG, 0, 0, 0, 0, 0, 0},
 
     {"compress", OPT_XTRA_COMPRESS,
-     "Compress individual backup files using the "
-     "specified compression algorithm. Currently the only supported algorithm "
-     "is 'quicklz'. It is also the default algorithm, i.e. the one used when "
-     "--compress is used without an argument.",
+     "This option was deprecated starting with MariaDB 10.1.31 and 10.2.13 "
+	 "as it relies on the no longer maintained QuickLZ library.",
      (G_PTR *) &xtrabackup_compress_alg, (G_PTR *) &xtrabackup_compress_alg, 0,
      GET_STR, OPT_ARG, 0, 0, 0, 0, 0, 0},
 
     {"compress-threads", OPT_XTRA_COMPRESS_THREADS,
-     "Number of threads for parallel data compression. The default value is "
-     "1.",
+     "This option was deprecated with --compress ",
```

Comment:
Hi Gulshan!

Thanks for the patch! I have a couple quick notes:

1.  I don't think we want to get rid of the existing description text; I'd imagine we should just extend it with the deprecation message so we don't lose anything.

 2. I think the end part mentioning "with --compress" is a little ambiguous too. I'd suggest omitting it altogether, or replacing it with the "as it relies on the no longer maintained QuickLZ library." to be explicit (and the same goes for `--chunk-size` and `decompress`).

Reply:
@bnestere thanks for you suggestions, i will make the changes.